2021/index.html

```diff
@@ -1,33 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
```

Comment:
Wenn es keinen dringenden Grund gibt, bitte nicht einfach Dateien löschen. Im Allgemeinen ist es auch guter Stil, nicht jede Menge unabhängige Änderungen, sondern thematisch fokussierte und kleine Commits zu machen.

Reply:
Die Pull-Request beinhaltet die Jahresanzeige. Damit ist der Order von letztem Jahr überflüssig.
Ich habe auch gestern angefangen, das ganze in ne lokale IDE zu ziehen.